src/js/components/Layout/HeaderStyles.scss

```diff
@@ -130,15 +126,21 @@
 }
 
 .navbar-icons {
-  padding: 8px 16px 8px 16px;
-
```

Comment:
my reasoning was to try and use existing bootstrap classes instead of creating new styles where it is possible.
I can change the bootstrap paddings to a more static eg. 28px it's not a problem